content/tutorials/use-simple-xrp-payments/send-xrp.md

```diff
@@ -0,0 +1,593 @@
+- That you are connected to a server on the test network.
+
+See the full list of [transaction results](transaction-results.html) for more possibilities.
+
+
+<div class="interactive-block" id="interactive-submit">
+<div class="interactive-block-inner">
+  <div class="breadcrumbs-wrap">
+    <ul class="breadcrumb prereqs">
+      <li class="breadcrumb-item active bc-generate"><a href="#interactive-generate">Generate</a></li>
+      <li class="breadcrumb-item disabled bc-connect"><a href="#interactive-connect">Connect</a></li>
+      <li class="breadcrumb-item disabled bc-prepare"><a href="#interactive-prepare">Prepare</a></li>
+      <li class="breadcrumb-item disabled bc-sign"><a href="#interactive-sign">Sign</a></li>
+      <li class="breadcrumb-item disabled current bc-submit"><a href="#interactive-submit">Submit</a></li>
+      <li class="breadcrumb-item disabled bc-wait"><a href="#interactive-wait">Wait</a></li>
+      <li class="breadcrumb-item disabled bc-check"><a href="#interactive-check">Check</a></li>
+    </ul><!--/.breadcrumb.prereqs-->
+  </div><!--/.breadcrumbs-wrap-->
+  <button id="submit-button" class="btn btn-primary connection-required"
+    title="Connection to Test Net required" disabled>Submit
+    example transaction</button>
+    <div id='loader-{{n.current}}' style="display: none;"><img class='throbber' src="assets/img/rippleThrobber.png"></div>
+    <div id="submit-output"></div>
+</div>
+</div>
+<script type="application/javascript">
+  $("#submit-button").click( async function() {
+    $("#submit-output").html("") // Wipe previous output
+    $("#loader-{{n.current}}").show()
+
+    const txBlob = $("#signed-tx-blob").text()
+    const earliestLedgerVersion = await api.getLedgerVersion()
+    $("#earliest-ledger-version").text(earliestLedgerVersion)
+
+    try {
+      const result = await api.submit(txBlob)
+      $("#loader-{{n.current}}").hide()
+      $("#submit-output").html(
+        "<div><strong>Tentative result:</strong> " +
+        result.resultCode + " - " +
+        result.resultMessage +
+        "</div>"
+      )
+
+      // Update all breadcrumbs & active next step
+      $(".bc-submit").removeClass("active")
+      $(".bc-submit").addClass("done")
+      $(".bc-wait").removeClass("disabled")
+      $(".bc-wait").addClass("active")
+    }
+    catch(error) {
+      $("#loader-{{n.current}}").hide()
+      $("#submit-output").text("Error: "+error)
+    }
+
+  })
+</script>
+
+### {{n.next()}}. Wait for Validation
+
+Most transactions are accepted into the next ledger version after they're submitted, which means it may take 4-7 seconds for a transaction's outcome to be final. If the XRP Ledger is busy or poor network connectivity delays a transaction from being relayed throughout the network, a transaction may take longer to be confirmed. (For information on how to set an expiration for transactions, see )
+
+You can trigger code to run using the `ledger` event type in RippleAPI. For example:
```

Comment:
A little bit confused by this sentence. Maybe adding what running the code will do -- will help? Humbly suggested edit:

> You can run code that uses the RippleAPI [`ledger` event type] to signal whenever a new ledger version is validated on the connected server.

Reply:
Rephrased to:

> You use the `ledger` event type in RippleAPI to trigger your code to run whenever there is a new validated ledger version.